tests/netperf_test.py

```diff
@@ -65,24 +70,37 @@ def run_netperf_parse(self, device, timeout=60):
     def run_netperf(self, device, ip, opts="", timeout=60):
         self.run_netperf_cmd(device, ip, opts=opts)
         return self.run_netperf_parse(device)
+   
+    def kill_netserver(self, device):
+        device.sendline("kill -9 `pidof netserver`")
+        device.expect(prompt)
 
     def runTest(self):
         super(NetperfTest, self).runTest()
 
         board.sendline('mpstat -P ALL 30 1')
-        speed = self.run_netperf(lan, "192.168.0.1 -c -C -l 30")
-        board.expect('Average.*idle\r\nAverage:\s+all(\s+[0-9]+.[0-9]+){10}\r\n')
+        board.expect('Linux')
+
+	wan_eth1_addr = wan.get_interface_ipaddr(self.iface_dut)
```

Comment:
wan.iface_dut not self.iface_dut

Reply:
used wan.gw instead of  wan.get_interface_ipaddr(self.iface_dut)